mne/io/pick.py

```diff
@@ -937,6 +944,8 @@ def _pick_data_or_ica(info, exclude=()):
 def _picks_to_idx(info, picks, none='data', exclude='bads', allow_empty=False,
                   with_ref_meg=True, return_kind=False):
     """Convert and check pick validity."""
+    # import pdb
```

Comment:
remove

Reply:
These should all be removed now. I kept open all the files I was in and checked for these.

Let me know if you catch more.